templates/admin/faq/_partials/card-question.html.twig

```diff
@@ -0,0 +1,42 @@
+<section class="suggestions-section">
+    <div class="suggestions-title-container">
+        <h5 class="suggestions-title">{{ 'faq-suggestions' | trans }}</h5>
+    </div>
+    <div class="row d-flex justify-content-center">
```

Comment:
Essa row for ade um container está fazendo o conteúdo da página vazar para a direita, criando um scroll horizontal, acho que resolveria colocando toda essa div dentro de um container, ou removendo a row e colocando um gap para afastar os cards